pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -6012,6 +6049,8 @@ def process_data(self):
                 sweep_points, self.proc_data_dict['sweep_points_dict'][qbn][
                     'cal_points_sweep_points']])
 
+            self.get_num_cal_points()
```

Comment:
Hasn't this already been called in process_data? It might help to add a comment why this needs to be called again.

Reply:
I now cleaned this up and fixed bugs.